### DISCUSSION TITLE
Idea for bookmarklets

### DISCUSSION THREAD

**Discussion by AnOrangeCone** [3kh0/ext-remover#1021] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I might be wrong but it seems like it would be possible to make a proxy with a built in javascript exec by making a text box useable with a keybind that injects your javascript code into the page on the proxy I want to try and make something like this but I don't know where to start or if it's possible. I will try to make a mockup or something and send it if I ever do it.

**Comment by akabutnicer** [comment]
First day back and I get posts like this. *Sigh*.
If an iframe of the proxy is being injected you just do `iframe.contentWindow.eval(script)`;.
THERE WE GO PROBLEM SOLVED 😱  Aren't I so intelligent?
